docker/puppetserver/Dockerfile

```diff
@@ -66,6 +66,6 @@ COPY docker/puppetserver/logback.xml \
      docker/puppetserver/request-logging.xml \
      /etc/puppetlabs/puppetserver/
 COPY docker/puppetserver/puppetserver.conf /etc/puppetlabs/puppetserver/conf.d/
-COPY docker/puppetserver/puppetdb.conf /etc/puppetlabs/puppet/
```

Comment:
Bug introduced in https://github.com/underscorgan/puppetserver/commit/361a9a155b6a2a532aab403d055faf3ebddcc185#diff-0bc70a0d801716e2c729708b9e6015baR58

Reply:
Sorry about that everyone!